File: data-jdbc-h2/src/main/java/com/example/data/jdbc/h2/DataJdbcH2Application.java
package com.example.data.jdbc.h2;

import org.springframework.aot.smoketest.thirdpartyhints.HikariRuntimeHints;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ImportRuntimeHints;

@SpringBootApplication
@ImportRuntimeHints(HikariRuntimeHints.class)
public class DataJdbcH2Application {

	public static void main(String[] args) throws InterruptedException {
		SpringApplication.run(DataJdbcH2Application.class, args);
		Thread.currentThread().join(); // To be able to measure memory consumption
	}

}

File: data/data-jdbc-h2-kotlin/src/main/kotlin/com/example/data/jdbc/h2/kotlin/DataJdbcH2Application.kt
package com.example.data.jdbc.h2.kotlin

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class DataJdbcH2Application

fun main(args: Array<String>) {
	SpringApplication.run(DataJdbcH2Application::class.java, *args)
}
